lisa/sut_orchestrator/azure/transformers.py

```diff
@@ -643,7 +660,7 @@ def _internal_run(self) -> Dict[str, Any]:
             runbook.gallery_image_osstate,
             runbook.gallery_image_hyperv_generation,
             runbook.gallery_image_architecture,
-            runbook.gallery_image_securitytype,
+            features,
```

Comment:
The property bag style parameters are not validated by schema, it's hard to find out supported values.

Reply:
It is not passed in through the schema, it is collected automatically from the marketplace image. I believe all feature values for a marketplace image are supported for a SIG image.